onnxruntime/python/tools/quantization/onnx_model.py

```diff
@@ -124,9 +203,11 @@ def get_parent(self, node, idx, output_name_to_node=None):
         return output_name_to_node[input]
 
     def find_node_by_name(self, node_name, new_nodes_list, graph):
-        """
-        Find out if a node exists in a graph or a node is in the
-        new set of nodes created during quantization. Return the node found.
+        """Find out if a node exists in a graph or a node is in the
+        new set of nodes created during quantization.
+
+        Return:
```

Comment:
nit: https://google.github.io/styleguide/pyguide.html#doc-function-returns

```suggestion
        Returns:
```